.changeset/friendly-horses-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wpengine/wp-graphql-content-blocks": patch
+---
+
+Chore: Changeset
```

Comment:
@theodesp We should probably make this more descriptive as it doesn't describe the changes related to `CoreSeparator`